lhotse/cut.py

```diff
@@ -78,31 +83,34 @@ def truncate(
             self,
             *,
             offset: Seconds = 0.0,
-            until: Optional[Seconds] = None,
-            keep_excessive_supervisions: bool = True
+            duration: Optional[Seconds] = None,
+            keep_excessive_supervisions: bool = True,
+            preserve_id: bool = False
     ) -> 'Cut':
         """
-        Returns a new Cut that is a sub-region of the current Cut. The `offset` parameter controls the start of the
-        new cut relative to the current Cut's start, and `until` parameter controls the new cuts end.
-        Since trimming may happen inside a SupervisionSegment, the caller has an option to either keep or discard
-        such supervisions with `keep_excessive_supervision` flag.
-        Note that no operation is done on the actual features - it's only during load_features() when the actual
-        changes happen (a subset of features is loaded).
-
-        Example:
-        >>> from math import isclose
-        >>> cut = Cut(id='x', channel=0, start=3.0, duration=8.0, features='dummy', supervisions=[])
-        >>> trimmed_cut = cut.truncate(offset=5.0, until=7.0)
-        >>> trimmed_cut.start == 5.0 and isclose(trimmed_cut.duration, 2.0) and isclose(cut.end, 7.0)
+        Returns a new Cut that is a sub-region of the current Cut.
+
+        Note that no operation is done on the actual features - it's only during the call to load_features()
```

Comment:
Just curious about something... what would it take to re-dump the features from such a manifest?  E.g. suppose you were planning to train for many epochs and did not want to incur repeated extra I/O?

Reply:
It should be fairly simple, one method + one CLI mode. I'll add an issue.